benchmarks/Microsoft.AspNetCore.Routing.Performance/Matchers/DfaMatcher.cs

```diff
@@ -1,353 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.Routing.Template;
-
-namespace Microsoft.AspNetCore.Routing.Matchers
-{
-    internal class DfaMatcher : Matcher
```

Comment:
These files got moved to the test project.

Reply:
You can treat the matchers and builders as a copy-paste. Nothing really worth reviewing there at this point, and I mostly copy-pasted them.